### DISCUSSION TITLE
Troubleshooting 404 Error in Nuxt.js with nuxtjs/i18n

### DISCUSSION THREAD

**Discussion by addda3786** [nuxt/nuxt#28150] [Questions (Nuxt 2)]
I am using Nuxt.js with the nuxtjs/i18n module. The i18n configuration has strategy set to prefix and defaultLocale set to en-US. The vueI18n configuration has legacy set to false and locale set to en-US.

When I access a specific page, let's say the path is /topics, the URL changes to /en-US/topics, but the content displayed is a 404 error page. It seems like the router is encountering a "page not found" issue. How can I identify and resolve the cause of this error?

**Comment by shaojiaqi08** [comment]
You need to configure the `locales ` property in the `i18n` object in `nuxt.config.ts`.
